css-four-column-layout/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Four Column Layout</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <header>How It Works</header>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-quarter">
+          <img class="screen" src="images/screen-1.png">
+        </div>
+        <div class="column-quarter">
+          <img class="screen" src="images/screen-2.png">
+        </div>
+        <div class="column-quarter">
+          <img class="screen" src="images/screen-3.png">
+        </div>
+        <div class="column-quarter">
+          <img class="screen" src="images/screen-4.png">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-quarter">
+          <img class="logo" src="images/switch-icon.png">
+        </div>
+        <div class="column-quarter">
+          <img class="logo" src="images/filter-icon.png">
+        </div>
+        <div class="column-quarter">
+          <img class="logo" src="images/slider-icon.png">
+        </div>
+        <div class="column-quarter">
+          <img class="logo" src="images/bars-icon.png">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-quarter">
+          <h3>App Switching</h3>
+        </div>
+        <div class="column-quarter">
+          <h3>Trip Filtering</h3>
+        </div>
+        <div class="column-quarter">
+          <h3>Service Filters</h3>
+        </div>
+        <div class="column-quarter">
+          <h3>Stats + Mileage</h3>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-quarter">
+          <p>Toggles all your apps on and off.</p>
+        </div>
+        <div class="column-quarter">
+          <p>Evaluates your trip requests.</p>
+        </div>
+        <div class="column-quarter">
+          <p>Accepts only your ideal trips.</p>
+        </div>
+        <div class="column-quarter">
+          <p>Tracks your trips across all apps.</p>
+        </div>
+      </div>
+    </div>
```

Comment:
All of this content is related, so it should not be separated into individual rows. You want one row with four columns, each of which holds the phone image, icon, and text associated with that image and icon.